src/powershell/enable-rdp.ps1

```diff
@@ -28,3 +28,48 @@ if ($(Get-NetFirewallRule -Name $name).Enabled -ne $true) {
     Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
 }
 Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Allow Terminal Server shadowing through the Windows firewall
+$name = "Remote Desktop - Shadow (TCP-In)"
+Write-Output "[ ] Configuring firewall rule: $name"
+Set-NetFirewallRule -DisplayName $name -Enabled True -Profile Private
+if ($(Get-NetFirewallRule -DisplayName $name).Enabled -ne $true) {
+    Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
+}
+Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Enable Terminal Server shadowing
+# Set value to option 2:
+# "Configure Group Policy setting to not require the user’s consent"
+$name = "Shadow"
+$path = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp'
+Write-Output "[ ] Configuring setting: $name"
+Set-ItemProperty -Path $path -Name $name -Value "2"
+if ($(Get-ItemProperty -Path $path -Name $name).Shadow -ne 2) {
+    Write-Error "[X] Failed to verify setting: $name" -ErrorAction Stop
+}
+Write-Output "[*] Setting successfully verified: $name"
+
+# Give the Administrator account full control of Remote Desktop Services
```

Comment:
As the `BUILTIN\Administrators` group is not the same thing as the built-in Administrator account.

```suggestion
# Give the Administrators group full control of Remote Desktop Services
```

Reply:
@mcdonnnj resolved with [881105e](https://github.com/cisagov/windows-server-packer/pull/22/commits/881105ef37628dc8692092ee78cd3eda1588861a)